pkg/relayer/proxy/async.go

```diff
@@ -96,3 +98,62 @@ func (server *relayMinerHTTPServer) handleAsyncConnection(
 
 	return nil
 }
+
+// forwardAsyncConnection instantiates two websocket connections that:
```

Comment:
Can you #PUC why we need two websockeet connections and one is not enough?

Reply:
It is written in the comment ([103 & 104](https://github.com/pokt-network/poktroll/blob/647cf9d53a761ca0fda0779f12800ed95f89d16c/pkg/relayer/proxy/async.go#L103-L104)).

I need two connections to proxy the websocket traffic from the client to the backend websocket service. One connection (clientConn) between your proxy and the client, and one connection (serviceConn) between your proxy and the backend service.